packages/web-server/package.json

```diff
@@ -8,9 +8,19 @@
     "directory": "packages/web-server"
   },
   "license": "MIT",
-  "main": "./dist/server.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
```

Comment:
```suggestion
      "default": "./dist/webServer.js"
```

Reply:
Done in https://github.com/redwoodjs/redwood/pull/9884/commits/7a0d694c0bd3d99398f3fa6e4e0a72173c7f6bcc